src/frontend/planner_test/tests/testdata/output/basic_query.yaml

```diff
@@ -42,6 +42,15 @@
     StreamMaterialize { columns: [v1, t._row_id(hidden)], stream_key: [t._row_id], pk_columns: [t._row_id], pk_conflict: "NoCheck" }
     └─StreamFilter { predicate: (t.v1 < 1:Int32) }
       └─StreamTableScan { table: t, columns: [t.v1, t._row_id], pk: [t._row_id], dist: UpstreamHashShard(t._row_id) }
+- sql: |
+    create table t (v1 int, v2 int, v3 int);
+    select * except (v1, v2) from t;
```

Comment:
Can we add more tests which explicitly select `except` columns?
Something like:
```sql
select v3 from (select * except (v1, v2) from t
```

Can we also add tests which have selected the excepted row, and re-select already included rows?
```
select * except (v1, v2), v1 from t; -- v1 excluded from except.
select * except (v1, v2), v3 from t; -- v3 included from except.
```